client/style.css

```diff
@@ -435,7 +455,7 @@ h1 {
   background: #f3e4e5;
 }
 .card li.correct {
-  background: #e4f3e4;
+  background: #e4f3e4 !important;
```

Comment:
Don't use `!important`. Try and find a more specific selector if you're trying to override something

Reply:
Done correctly now.  Also corrected another !important a few lines above and organized a few of these related css items together.